src/program/services/downloaders/__init__.py

```
@@ -1,70 +1,91 @@
     NoMatchingFilesException,
     NotCachedException,
     TorrentContainer,
     TorrentInfo,
+    UserInfo,
+)
+from program.services.downloaders.shared import (
+    DownloaderBase,
+    sort_streams_by_quality,
+    parse_filename,
 )
 from RTN import ParsedData
```

Ruff (0.14.6)

Redefinition of unused `ParsedData` from line 3

Remove definition: `ParsedData`

(F811)